### DISCUSSION TITLE
Arch Enemies First Playtest

### DISCUSSION THREAD

**Discussion by mango-gremlin** [mango-gremlin/arch-enemies#306] [Playtest Discussion 0.1]
The first playable release of our game! Yay!

<hr /><em>This discussion was created from the release <a href='https://github.com/mango-gremlin/arch-enemies/releases/tag/v.0.1.0-alpha-playtest'>Arch Enemies First Playtest</a>.</em>

**Comment by Kabooni** [comment]
Feedback von zwei Leuten:
- Tutorial generell zu schwer und zu wenig erklärt. Man wollte immer Tiere direkt zurückziehen, alle buttons wurden komplett ignoriert und müssen unbedingt auch erklärt werden
- hazard wurde nicht gesehen /verstanden
- snapping nicht intuitiv, es wurde immer versucht die animals selber zu platzieren und es wurde nicht auf den Mauszeiger geachtet
- Fox schwebt in der Luft -> Ast wird nicht gesehen

- Dialog sehr cool, aber Rechtschreibfehler und man drückt immer ausversehen zuerst auf den exit button
- spider und squirrel icon nicht sofort erkennbar als die jeweiligen Tiere (Besonders squirrel grau verwirrt Leute)
- y sort funktioniert nicht immer, besonders nicht auf der Froschinsel. Auch sollte man nicht durch Tiere durchlaufen können
- Die Eingänge der horizontalen Brücken sind zu eng

Froschlevel Lösung
![grafik](https://github.com/mango-gremlin/arch-enemies/assets/57258671/d5f3bf93-2a60-43bd-9f24-e1a845dc7790)

Final level Lösung
![grafik](https://github.com/mango-gremlin/arch-enemies/assets/57258671/41835553-9258-4f2a-95a1-2c2a54b72f2e)


-> Tutoriallevel nicht intuitiv und zu schwer, aber Musik, design und art sehr sehr cool. Die Level sind gut lösbar und der generelle gameloop hat funktioniert, nur es braucht viel mehr Erklärung am Anfang

**Comment by Raining-Cloud** [comment]
Feedback paar aus ner anderen Gruppe:

- lauf sound kommt nicht gut an (zu laut und zu hart), buggt auch teilweise beim öffnen von menüs während dem laufen
- y-sort von sehr vielen Sachen ist weird (übers Reh laufen, dirt bei bridge oben)
- Quest-Buttons gehen nicht weg => was solved already text verschwindet nicht
- Ol'Nutter wird geliebt
- overworld movement ist komisch priorisiert
- snapping von animals in bridges ist am anfang nicht klar

**Reply by MettBrotDev** [reply]
![image](https://github.com/mango-gremlin/arch-enemies/assets/116958361/eb57c519-babb-4d29-b0c1-d68ad99c4afd)
Unsere Lösung .-.
![image](https://github.com/mango-gremlin/arch-enemies/assets/116958361/6d4728ea-0333-42c9-8d25-9db3d328962f)
(Passiert wenn man das Spiel schließt und wieder öffnet, Spieler Koordinaten werden gespeichert aber der Spielstand nicht)

**Reply by MettBrotDev** [reply]
![image](https://github.com/mango-gremlin/arch-enemies/assets/116958361/ee4ed58b-9bb3-41e3-a2e8-35dbd2f73d2c)
Ah und noch zum Feedback dazu, dass man solche Schrägen nicht hochlaufen kann fühlt sich komisch an.

**Comment by marcel-iv** [comment]
Feedback einer Person:

Kritik:
- Text in den Boxen im Tutorial sollte mehr Abstand zum Rand haben
- Rechtschreibfehler (Deer Intro Box, Ol' Nutter Dialog)
- Nach Streit am Anfang nimmt man an, dass das schon der ganze Dialog mit den Tiere war -> extra Sprite für Tierdiskussion/Sprites davor unsichtbar machen?
- Man sollte nicht durch NPCs durchlaufen können
- Möglichkeit platzierte Tiere zu entfernen wäre sehr gut, zurückdraggen zur Box war der intuitivste Vorschlag
- Wenn man nichts dragged, dann spawned der Grid -> nervig
- Clipping probleme in der Overworld, teilweise inconsistant durch welche Sachen man durchlaufen kann
- Bei der Intro Cutscene wurde der Damm zuerst nicht als solcher interpretiert
- Was beim Neustarten des Spiels vom Save File geladen wird, und was nicht, ist sehr unintuitiv
- Möglichkeit neuen Spielstand anzufangen ohne manuell das file zu löschen
- Teilweise konnte man Items doppelt aufheben? 
- Im letzten Level wärs cool wenn die Dornen rechts oben etwas schmaler wären, damit man alternativ auch drüber springen kann
- Wär cool wenn man Level replayen könnte ohne komplett neu zu starten
- Wär cool wenn man nach betreten der Victory Zone das Level noch editieren könnte (neuer Knopf -> Goal Menu unsichtbar, player_reset)
- Spider-Tooltip sollte auch beim letzten Level angezeigt werden, im Fall dass man es zuerst betritt 

Sehr positiv angemerkt wurde:
- Art allgemein, besonders die Frösche (Merch wo?)
- Dialog/Charakterisierung, auch in Tooltips und Cutscene
- Tooltips allgemein
- Sound und Lautstärke (bei ~30% Volume)
- Replayability von Leveln um effizientesten Weg zu finden

**Comment by Raffael-Rack** [comment]
Feedback von meinen Eltern:

Sachen die Verbessert werden könnten:

1. Der Damm in der Intro cutscene ist nicht gut erkennbar (50% erkennungsrate)
2. Startmenü ist nicht mit der scroll alignt.
3. Laufgeräusch ist zu laut
4. Spacing von den Worten im tutorial, sie überschneiden sich teilweise mit dem Rand.
5. Mehr Tutorial (was bedeuten die farben, was sind die Hazards, wo muss man überhaupt hin, fox & dragmode)
6. Tier placement ist nicht immer intuitiv, weil der mauszeiger außerhalb des sprites ist.
7. Aktionen im dialog besser kenntlich machen. Die < aktionspfeile > wurden für scrollbuttons gehalten.
8. Auch im dialog: Next und Previous button tauschen, damit man immer an die selbe stelle klicken muss, buttons gleichgroß machen, exit button in die ecke tun.
9. Button prompts sollten verschwinden nachdem man die Quest eines NPC/ein level beendet hat.
10. Beim Deer wurde der "help everyone before I help you" Dialog angezeigt obwohl allen bereits geholfen wurde.
11. Die laufanimation des fuchses wenn er sich nach vorne/unten auf der overworld bewegt.

**Comment by aylinfi** [comment]
Feedback was ich bekommen habe:

## Cutscene
- Damm ist eher nicht erkennbar
- hier vielleicht auch noch Zusammenhang mit Bibern und Fröschen zeigen

## Levels

- Tutorial hat die Grundlagen nicht gut genug erklärt
- Dornen und Frösche werden nicht als Hazards erkannt
- (vielleicht Hazard Hitboxen kleiner machen)
- Wo Tiere platziert werden wenn man loslässt ist sehr unintuitiv
- frustrierend dass Buttons im Foxmode zusehen aber nicht benutzbar sind
- blöd dass man Tiere nicht wieder aufheben kann
- Es ist sehr einfach aus Versehen auf "reset" anstatt "one step back" zu klicken (was sehr frustrierend sein kann)
- Spinnentutorial macht keinen Sinn wenn man Level auch ohne Spinnen starten kann
- Viele wussten nicht, dass sie erst neue Tiere brauchen um die Level lösen zu können 
- dazu auch nicht klar, dass man durchs Menü zurück in die Overworld kann
- Allgemein nicht unbedingt klar, dass man mehr Tiere kriegen kann

## Overworld
- Frosch/Streit kann übersehen/ignoriert werden
- Hirsch Dialog sehr verwirrend
- Schlange redet über Schwimmen -> wird missverstanden als game mechanic
- nicht klar das nach Quest Tiere hinzukommen, visuell Tieranzahl highlighten?
- Items verschwinden nicht aus Inventar wenn erwartet
- dialog next/previous tauscht was verwirrend ist
- "Was solved already" Text ist nicht mehr verschwunden
- Wunsch dass im Dialog wichtige Dinge hervorgehoben werden
- Endszene wurde gewünscht

**Reply by aylinfi** [reply]
- Außerdem konnte der Fuchs im Finallevel den oberen Levelrand verlassen
- und es wurde sich gewünscht, dass man die Schlange rotieren/spiegeln kann

**Comment by mango-gremlin** [comment]
Vielen Dank für alle Rückmeldungen und Playtester!!!!
Wie man sicher schon sieht wurde jetzt das feedback aufgenommen und der plan ist eigentlich alles davon irgendwie zu fixen/addressieren; das meiste wurde auch schon gefixed. am 2.1. wird es dann einen final release geben.
